pkgs/applications/graphics/antimony/default.nix

```diff
@@ -23,7 +23,7 @@ in
     '';
 
     buildInputs = [
-      libpng python3 (boost.override { python = python3; })
+      libpng python3Packages.python python3Packages.boost
```

Comment:
use python3.pkgs.python and python3.pkgs.boost. No need to make the change everywhere.